src/DoctrineBatchUtils/BatchProcessing/SimpleBatchIteratorAggregate.php

```diff
@@ -17,28 +16,46 @@
 use function is_object;
 use function key;
 
+/**
+ * @template A
```

Comment:
TKey, TValue instead of A, B

Reply:
I think `A` should be constrained to `int|string`

Reply:
same as https://github.com/Ocramius/DoctrineBatchUtils/pull/99#discussion_r551004131